File: 프로그래머스/0/181938. 두 수의 연산값 비교하기/두 수의 연산값 비교하기.py
def solution(a, b):
    answer = 0
    
    str_ab = str(a) + str(b)
    str_ba = str(b) + str(a)
    
    ab = 2 * a * b
    
    
    if int(str_ab) > ab:
        answer = int(str_ab)
    elif int(str_ab) < ab:
        answer = ab
    elif int(str_ba) > ab:
        answer = int(str_ba)
    elif int(str_ba) < ab:
        answer = ab
    
    return answer

File: 프로그래머스/0/181938. 두 수의 연산값 비교하기/두 수의 연산값 비교하기.java
class Solution {
    public int solution(int a, int b) {
    
        String str_ab = Integer.toString(a) + Integer.toString(b);
        int num_ab = Integer.parseInt(str_ab);
        int sum = 2 * a * b;
        
        if(num_ab > sum){
            return num_ab;
        }
        else{
            return sum;
        }
    }
}